sparse-index.c

```diff
@@ -261,7 +261,7 @@ void ensure_full_index(struct index_state *istate)
 			warning(_("index entry is a directory, but not sparse (%08x)"),
 				ce->ce_flags);
 
-		/* recursively walk into cd->name */
+		/* recursively walk into ce->name */
```

Comment:
Since this typo got into upstream, do you maybe want to _not_ mark this as a `fixup!`?

Reply:
That particular `fixup!` commit is more of a reminder to myself to include it in a cleanup patch somewhere down the line (not in this series)